test/runMultipart.test.js

```diff
@@ -0,0 +1,141 @@
+'use strict'
+
+const test = require('tap').test
+const { tmpdir } = require('os')
+const { join } = require('path')
+const { writeFile } = require('fs')
+const { promisify } = require('util')
+const run = require('../lib/run')
+const helper = require('./helper')
+const server = helper.startMultipartServer()
+const writef = promisify(writeFile)
+
+test('run should retunr an error with invalid form options', async t => {
+  const cases = [
+    {
+      name: 'invalid JSON',
+      value: 'u',
+      message: 'Invalid JSON or file where to get form data'
+    },
+    {
+      name: 'non existing json file',
+      value: 'nonexisting.json',
+      message: 'Invalid JSON or file where to get form data'
+    },
+    {
+      name: 'missing path key in file type',
+      value: '{ "image": { "type": "file" }}',
+      message: 'Missing key \'path\' in form object for key \'image\''
+    },
+    {
+      name: 'missing value in text type',
+      value: '{ "image": { "type": "text" }}',
+      message: 'Missing key \'value\' in form object for key \'image\''
+    },
+    {
+      name: 'missing value in text type',
+      value: '{ "image": { "type": "random" }}',
+      message: 'A \'type\' key with value \'text\' or \'file\' should be specified'
+    }
+  ]
```

Comment:
I think we should support an API where the form value is a JS object, and not just the json version of it